storm-core/src/jvm/org/apache/storm/utils/Utils.java

```diff
@@ -2092,6 +2092,11 @@ public static String addToClasspath(String classpath,
         return _instance.addToClasspathImpl(classpath, paths);
     }
 
+    public static String addToClasspath(Collection<String> classpaths,
+                Collection<String> paths) {
+        return _instance.addToClasspathImpl(classpaths, paths);
+    }
+
```

Comment:
Since we are making this static method mockable anyway, why don't we add a simple test?